doc/ROS2/setup/Webots.md

```diff
@@ -7,6 +7,14 @@ Follow the official installation guide for Webots on Linux with APT on the
 > **_NOTE:_**
 Some of the steps result in larger package downloads.
 
+Set environment variables for Webots home directory, the Webots controller library directory and add the Webots controller library to linker search path.
```

Comment:
Add a sentence that setting them in ".bashrc" is recommended to have it in any shell automatically. It reads as it is a "must" right now.  

Some users are picky regarding changes in this file and prefer other means to set this up.